Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from fastapi import FastAPI\r\nfrom core import models\r\nfrom core.database import engine\r\nfrom endpoints import users, admin\r\n\r\nmodels.Base.metadata.create_all(bind=engine)\r\n\r\napp = FastAPI()\r\napp.include_router(users.router)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 4067c578a46dce512d13909a2b681d08e7baf3cd)
+++ b/main.py	(date 1664280780870)
@@ -5,5 +5,5 @@
 
 models.Base.metadata.create_all(bind=engine)
 
-app = FastAPI()
+app = FastAPI()sss
 app.include_router(users.router)
